client/checkout/woopay/express-button/woopay-express-checkout-button.js

```diff
@@ -39,23 +39,23 @@ export const WoopayExpressCheckoutButton = ( {
 	useEffect( () => {
 		if ( ! isPreview ) {
 			wcpayTracks.recordUserEvent(
-				wcpayTracks.events.PLATFORM_CHECKOUT_EXPRESS_BUTTON_OFFERED,
+				wcpayTracks.events.WOOPAY_EXPRESS_BUTTON_OFFERED,
```

Comment:
I'd leave `checkout` word in this constant:
```suggestion
				wcpayTracks.events.WOOPAY_EXPRESS_CHECKOUT_BUTTON_OFFERED,
```